ros2controlcli/ros2controlcli/verb/set_hardware_component_state.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023 ROS2Control Developer Team
```

Comment:
```suggestion
# Copyright 2023 ros2_control Development Team
```

Reply:
If not, we should change all other to this or something similar that looks nicer, but `ros2_control` has to be written correctly.

Reply:
I think I just copied that from some other file. fine for me!